content/blog/2020-08-04-osf.md

```diff
@@ -0,0 +1,86 @@
+description: >
+  An introduction to the Open Science Framework (OSF).
+twitterImg: blog/2019/06/04/post-template/name-of-image.png
+output:
+  html_document:
+    keep_md: yes
+---
+
+[osfr][osfr-gh] provides a (*hopefully*) convenient R interface to the [Open Science Framework][osf] (aka, OSF). The package completed its rOpenSci peer-review earlier this year and has been [available on CRAN][osfr-cran] since February.
+
+Throughout its development and since its release I've had numerous conversations with members of the R community about OSF (and osfr), and through these interactions a couple recurring patterns emerged. First, it seems that many R users have heard of OSF but relatively few have first-hand experience with it. Second, I'm often asked how OSF compares to GitHub and whether it would even be useful for someone who already uses GitHub to manage their research.
+
+In a future post I'll highlight some features of osfr and demo some real world use cases. However, in order to extract any value from the package, you need to be an OSF user. And so, I wanted to take this opportunity to provide a little background about OSF, what it offers, how it differs from GitHub, and where it might fit into your workflow as an R/GitHub user.
+
+Before diving in, I do want to acknowledge that OSF is pretty multi-faceted, and includes a number of other services outside of its core project management functionality, including things like [pre-print servers][osf-preprint] and a [research registration repository][osf-registry]. While incredibly cool and noteworthy, those aspects of OSF fall outside the scope of this post, which I'm limiting to project management features.
+
+### Is OSF like a GitHub for research?
+
+It's not a perfect analogy as the two services differ in important ways that I'll discuss below, but if you're already familiar with GitHub it certainly provides a useful mental model to get a sense of what OSF offers. For example, like GitHub, OSF is a hosted project management service organized around project repositories that can be private, shared with select collaborators, or made publicly accessible. OSF projects also include cloud storage for uploading project files, automatic file versioning, markdown-based wikis for documentation and/or shared task lists, traffic analytics to assess impact and engagement, and they can even be "forked" by other users.
+
+### How are they different?
+
+Ultimately the two services are different because they are designed for different purposes: facilitating software development vs facilitating scientific research. While the line separating these tasks is often blurry (or non-existant), supporting research requires accommodating files of all shapes and sizes, whereas software (generally) comes in one flavor: plain text. Furthermore, GitHub is intended for software developers, an inherently technical userbase, whereas OSF is designed to accomodate users with a much wider range of computational expertise.
```

Comment:
non-existent
user base
accommodate
```suggestion
Ultimately the two services are different because they are designed for different purposes: facilitating software development vs facilitating scientific research. While the line separating these tasks is often blurry (or non-existent), supporting research requires accommodating files of all shapes and sizes, whereas software (generally) comes in one flavor: plain text. Furthermore, GitHub is intended for software developers, an inherently technical user base, whereas OSF is designed to accommodate users with a much wider range of computational expertise.
```